web-components/src/components/inputs/new/ImageDrop/ImageDrop.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { forwardRef, useState } from 'react';
```

Comment:
It might make sense to rename this to `FileDrop`
Also I've just realized it might not handle multiple files being dropped as expected, in which case it should `renderModal` for those files successively (https://www.figma.com/file/Bksz1JeDYxQVIXdI46EgPT/Project-Posts?type=design&node-id=859-70764&mode=design&t=iX8T1kr40x22X6r2-0)